framework/test/mocha/integration/rounds.js

```diff
@@ -258,6 +258,35 @@ describe('rounds', () => {
 		return accounts;
 	}
 
+	function recalculateVoteWeightReceived(_accounts, voters) {
+		const accounts = _.cloneDeep(_accounts);
+
+		// Reset voteWeightReceived for all accounts
+		_.each(accounts, account => {
```

Comment:
do we need to use `_.each` here?

Reply:
Not really. Just did `recalculateVoteWeight` based on `recalculateVote`

Reply:
Let's refactor on the next iteration